articles/cosmos-db/how-to-provision-container-throughput.md

```diff
@@ -94,6 +94,27 @@ offer.content.offerThroughput = 2000;
 await client.offer(offer.id).replace(offer);
 ```
 
+### <a id="dotnet-cassandra"></a>MongoDB API
```

Comment:
The id for this a tag conflicts with the cassandra one below. Should be changed to "dotnet-mongodb".